lib/ansiblelint/utils.py

```diff
@@ -68,9 +68,12 @@ def initialize_logger(level=0):
     handler = logging.StreamHandler()
     formatter = logging.Formatter('%(levelname)-8s %(message)s')
     handler.setFormatter(formatter)
-    _logger.addHandler(handler)
+    logger = logging.getLogger("ansiblelint")
+    logger.addHandler(handler)
     # unknown logging level is treated as DEBUG
-    _logger.setLevel(VERBOSITY_MAP.get(level, logging.DEBUG))
+    logging_level = VERBOSITY_MAP.get(level, logging.DEBUG)
+    logger.setLevel(logging_level)
+    logger.debug("Logging initialized to level %s", logging_level)
```

Comment:
Let's use `_logger` here in order to test that the config above works.
```suggestion
    _logger.debug("Logging initialized to level %s", logging_level)
```